frontend/webpage/src/router/index.js

```diff
@@ -17,7 +18,12 @@ const routes = [
   {
     path: '/about',
     name: 'About',
-    component: About,
+    component: About
+  },
+  {
+    path: '/searchroom',
+    name: 'SearchRoom',
```

Comment:
This will make so that you can have display the router link with an space on the toolbar
```suggestion
    name: 'Search Room',
```

Reply:
I remember trying that but it would crash the whole web app, I'll try again, maybe I remember it wrong.

Reply:
Fixed.